browser/main/lib/RcParser.js

```diff
@@ -0,0 +1,23 @@
+import _ from 'lodash'
+
+const path = require('path')
+const sander = require('sander')
+
+function parse (boostnotercPath) {
+  if (!sander.existsSync(boostnotercPath)) return
```

Comment:
This line failed when return void.
https://github.com/BoostIO/Boostnote/pull/318/files#diff-4bdfd7ba1c9cd532de6563d2e3cb4934R14